pkg/core/mpt/billet.go

```diff
@@ -11,6 +11,13 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/util"
 )
 
+// DummySTTempStoragePrefix is a dummy contract storage item prefix that may be
+// passed to Billet constructor when Billet's restoring functionality is not
+// used, i.e. for those situations when only traversal functionality is used.
+// Note that using this prefix for MPT restoring is a no-op since Billet must
```

Comment:
It's not a no-op, it's an error leading to panic to me.

Reply:
A configuration error in fact, yes.